### PR TITLE
Company page (Copy from https://github.com/kdti/kdti-client/pull/45)

### LINKED ISSUE #37: Create page to update company information

### PR DESCRIPTION
**Pull request by allanhal**
Addition of company's page to edit information about company in the admin.

It closes #37 
Copy from https://github.com/kdti/kdti-client/pull/45

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -2,709 +2,767 @@
 # yarn lockfile v1
 
 
-"@babel/code-frame@^7.0.0", "@babel/code-frame@^7.5.5":
```
Comment (carlosxjm) [inline yarn.lock]: No dependencies updated, no yarn.lock updates

File: pages/admin/company.vue [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+      states,
+    }
+  },
+  methods: {
+    check() {
+      const constraints = {
+        nameError: {
+          presence: { message: '^Nome da empresa não pode ser vazio' },
+        },
+        emailError: {
```
Comment (carlosxjm) [inline pages/admin/company.vue]: Remove error suffix

File: pages/admin/company.vue [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+              {{ stateElement.desc }}
+            </option>
+          </select>
+          <p class="error">{{ errors.stateError | first }}</p>
+          <label for="city">Cidade</label>
+          <input v-model="city" name="city" type="text" />
+          <p class="error">{{ errors.cityError | first }}</p>
+          <label for="neighborhood">Bairro</label>
+          <input v-model="neighborhood" name="neighborhood" type="text" />
+          <p class="error">{{ errors.stateError | first }}</p>
```
Comment (carlosxjm) [inline pages/admin/company.vue]: Remove error suffix

File: pages/admin/company.vue (hunk 2 above)
Comment (allanhal) [reply pages/admin/company.vue]: Was not able to. Can't say why :/

**Comment by carlosxjm** [comment]
Thanks @allanhal, thanks @douglastmp

**Pull request merged**